frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/OptionListSelector/OptionListSelector.test.tsx

```diff
@@ -70,7 +70,9 @@ describe('OptionListSelector', () => {
       },
     });
 
-    expect(await screen.findByText('Error')).toBeInTheDocument();
+    expect(
+      await screen.findByText(textMock('ux_editor.modal_properties_error_message')),
+    ).toBeInTheDocument();
```

Comment:
The test belonging to this expect can be removed when we never will render the returned error anyway. But Since removing this conditional error-rending is not actually a part of this PR (And I think it will be fixed in another PR we have), you can keep the original code for the error handling 🙈